SUPPORT.md

```diff
@@ -1,25 +1,27 @@
 # Aleph Support Policy
 
-*Technology is neither good nor bad; nor is it neutral.* (Kranzberg)
+_Technology is neither good nor bad; nor is it neutral._ (Kranzberg)
 
 The objective of the Aleph project is to provide powerful software to those who do investigative work in the public interest.
 
 **We develop this technology following the open source model, and will continue to release our code to the public. At the same time, we have chosen to limit the scope of the community to whom we will provide support and engagement.**
 
 The maintainers of this project limit support and responses both on GitHub and in the Slack channel to authorised groups and individuals. In order to receive authorisation, we require that you disclose the manner in which you use our technology. **We will decide if that use falls within the intended uses of Aleph.** Examples of intended uses could include:
 
-* Professional investigative journalists.
-* Activists, advocates and academics working in the public interest, and whose work is subject to an editorial policy
-* International bodies that have an investigative function.
+- Professional investigative journalists.
+- Activists, advocates and academics working in the public interest, and whose work is subject to an editorial policy
+- International bodies that have an investigative function.
 
 Support includes installation support, requests for new features or issues specific to your local Aleph installation. General bugs and contributions to the Aleph source code that can contribute to reliability of the system will be considered.
 
 Please submit a description of your use case along with your name, affiliation and email address using one of the following channels:
 
-* The Slack #intros channel
-* As part of the first GitHub issue that you file
-* Via our [contact form](https://requests.occrp.org/datadesk)
+- The Slack #intros channel
+- As part of the first GitHub issue that you file
+- Via our [contact form](https://requests.occrp.org/datadesk)
 
 Make sure you describe your goals, rather than the set of techniques that define your work (e.g. “investigations into human rights abuses in country X”, not “OSINT”). OCCRP is a non-profit organization. We do not offer commercial support or consulting services.
 
-August 2020
+The Aleph team supports feature versions for 12 months after the first major iteration of that version was released. For exmaple, we support Aleph 3.12 for 12 months after Aleph 3.12.0 was released. The Aleph team supports ugrades, but only from supported feature versions of the product.
```

Comment:
Maybe we should clarify what’s meant when we say "The Aleph team supports…"?

Pretty sure what’s meant here is that we can only provide support and help if you’re running a recent version of Aleph, but support can also have a broader meaning (e.g. backporting bug/security fixes to supported versions).